pages/bookmark.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles.css" />
+    <title>Quiz App</title>
+  </head>
+  <body>
+    <header>QuizApp for WebDevs</header>
+    <main>
+      <section class="card">
+        <p class="card__question">This should be a bookmarked question?</p>
+        <button class="card__button-answer">Show answer</button>
+        <button class="card__button-bookmark"></button>
+      </section>
+    </main>
+    <footer>
+      <nav>
+        <ul class="navbar">
+          <li><a href="../index.html" class="navbar--element">HOME</a></li>
+          <li>
+            <a href="./bookmark.html" class="navbar--element">BOOKMARK PAGE</a>
+          </li>
+          <li>
+            <a href="./profile.html" class="navbar--element">PROFILE</a>
+          </li>
+        </ul>
+      </nav>
+    </footer>
+  </body>
+</html>
+
+<!--TO DO: Make sure that only bookmarked questions are on this page.
+Update the navbar so that the bookmark icon is highlighted.-->
```

Comment:
you don't need javascript fot that. only change the background-color of the nav-item that is corresponding with the page it is on.